javascript/javascript1/week4/script.js

```diff
@@ -0,0 +1,131 @@
+let info = [{       // User name array
+    name: ""
+}]
+
+let todolist = []       // Todo list array
+
+function getReply(command) {
+    let arr = []        // Users inputs arrays
+    command1 = command.split(' ')
+    arr.push(command1)
+
+    switch (arr[0][0] + arr[0][1] + arr[0][2] + arr[0][3]) {
+
+        // Introduce inputs
+
+        case 'Hellomynameis':
+            if (info[0].name == arr[0][4]) {
+                console.log(`Okey okey i got it. Your name is ${info[0].name}. You dont need to repeat that.`)
+            }
+            else {
+                info[0].name = arr[0][4]
+                console.log(`Nice to meet you ${info[0].name}`)
+            }
+            break
+
+        // User name
+
+        case 'Whatismyname':
+            if (info[0].name == "") {
+                console.log('How can i know that. You dont introduce yourself!!!')
+            }
+            else {
+                console.log(`Your name is ${info[0].name}`)
+            }
+            break
+
+        // Set Timer
+
+        case 'Setatimerfor':
+            if (arr[0][5] == 'minutes') {
+                time = arr[0][4] * 60000
+            }
+            else if (arr[0][5] == 'seconds') {
+                time = arr[0][4] * 1000
+            }
+            else if (arr[0][5] == 'hours') {
+                time = arr[0][4] * 3600000
+            }
+            else { console.log('You write to wrong timer!!!') }
+             console.log(`Timer set for ${arr[0][4]} ${arr[0][5]}`)
+            setTimeout(function () {
+                console.log('Timer Done!!!');
+            }, time);
+            break
+
+        // Random number
+
+        case 'Sayanumberbetween' :
+            let min = Number(arr[0][4])
+            let max = Number(arr[0][6])
+            let random = min + Math.random() * (max-min)
```

Comment:
These should be const.

Reply:
Nice that you assign these values!
It's ok here, but normally it would be best to use full words for variables. It makes the code faster to read:
```js
const lowestNumber = Number(commandWords[4]);
const highestNumber = Number(commandWords[6]);
const randomNumber = lowestNumber + Math.random() * (highestNumber - lowestNumber);
```